searx/static/themes/simple/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "devDependencies": {
-    "eslint": "^8.11.0",
-    "grunt": "~1.5.2",
+    "eslint": "^8.18.0",
```

Comment:
We should also update eslint from root's `package.json`

https://github.com/searxng/searxng/blob/62e88e63a1d22c6d2298eec1aa1225801c0ca636/package.json#L3

Reply:
done